nina.css

```diff
@@ -0,0 +1,132 @@
+.container {
+  background: url(/photos/header-shapes\ mobile.png)  no-repeat;
+  background-size: cover;
```

Comment:
- [ ] After you implement the HTML Changes please add this image to the `<section>` tag you added instead of adding it to the whole div containing the header and the headline section.
and you can use this [link](https://www.w3schools.com/cssref/pr_background-position.asp) to help you position it right and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/165501675-df3cb4a2-0040-4342-a938-ef4ba379bbba.png)  | ![image](https://user-images.githubusercontent.com/68796232/165501350-3ee73d89-cfcc-4860-8f55-7388791878ec.png)  |